### PR TITLE
Complete unit-test-jest - Jimmy Tran

### PR DESCRIPTION
**Pull request by JimmyTran1206**
image

### CODE REVIEW

File: unit-tests-jest/numbers.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/**
+ * Modifies an object by multiplying the value of each key, but only if that
+ * value is a number. Key names are arbitrary.
+ */
+export function multiplyBy(
+  obj: Record<string, unknown>,
+  multiplier: number
+): Record<string, unknown> {
+  Object.entries(obj).forEach(([key, value]) => {
```
Comment (gerardoh13) [inline unit-tests-jest/numbers.ts:37]: This function will also work on arrays which can be parsed to strings. It should not.

**Review by gerardoh13 — APPROVED** [review]
Good job! You have very good coverage of test cases including edge cases.
